torch/_dynamo/decorators.py

```diff
@@ -179,9 +179,9 @@ def _apply_func_to_inner_tensors_of_same_dim(func, t, *args):
 
 
 @forbid_in_graph
-def mark_dynamic(t, index):
+def mark_dynamic(t, index, min=2, max=None):
```

Comment:
should `min` default to `None` as well, since the user might be confused if the pass `min=2` and it doesn't do anything different?

Reply:
Yes, default it to None

Reply:
done